tests/configs/standard/config.yaml

```diff
@@ -0,0 +1,352 @@
+        tss:x:59:
+        audio:x:63:
+        dbus:x:81:
+        screen:x:84:
+        nobody:x:99:
+        users:x:100:
+        systemd-journal:x:190:
+        systemd-network:x:192:
+        cgred:x:997:
+        ssh_keys:x:998:
+        input:x:999:
+    - name: dask-config
+      mountPath: /opt/lsst/software/jupyterlab/lsst_dask.yml
+      contents: |
+        # No longer used, but preserves compatibility with runlab.sh
+        dask_worker.yml: |
+          enabled: false
+    - name: idds-config
+      mountPath: /opt/lsst/software/jupyterlab/panda
+      modify: false
+      contents: |
+        # Licensed under the Apache License, Version 2.0 (the "License");
+        # You may not use this file except in compliance with the License.
+        # You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+        #
+        # Authors:
+        # - Wen Guan, <wen.guan@cern.ch>, 2020
+        [common]
+        # if logdir is configured, idds will write to idds.log in this directory.
+        # else idds will go to stdout/stderr.
+        # With supervisord, it's good to write to stdout/stderr, then supervisord can manage and rotate logs.
+        # logdir = /var/log/idds
+        loglevel = INFO
+        [rest]
+        host = https://iddsserver.cern.ch:443/idds
+        #url_prefix = /idds
+        #cacher_dir = /tmp
+        cacher_dir = /data/idds
+    # # -- Secrets for injection into user lab pods.  The name is again
+    # # the unique key that allows the Lab Controller to create the
+    # # contents.  Since these are stored as secrets, they will be created
+    # # from whole cloth, rather than modified from a base template.
+    # # If 'env' is specified, the mountPath will be stored in that environment
+    # # variable.
+    # secrets:
+    # - name: aws-credentials.ini
+    #   mountPath: /opt/lsst/software/jupyterlab/butler-secret/aws-credentials.ini
+    #   env: AWS_SHARED_CREDENTIALS_FILE
+
+    # - name: butler-gcs-idf-creds.json
+    #   mountPath: /opt/lsst/software/jupyterlab/butler-secret/butler-gcs-idf-creds.json
+    #   env: GOOGLE_APPLICATION_CREDENTIALS
+    # - name: butler-hmac-idf-creds.json
+    #   mountPath: /opt/lsst/software/jupyterlab/butler-secret/butler-hmac-idf-creds.json
+    # - name: postgres-credentials.txt
+    #   mountPath: /opt/lsst/software/jupyterlab/butler-secret/postgres-credentials.txt
+    #   env: PGPASSFILE
+  prepuller:
+    # -- config from sqr-066
+    config:
+      registry: lighthouse.ceres
+      docker:
+        repository: library/sketchbook
+      recommendedTag: recommended
+      numReleases: 1
+      numWeeklies: 2
+      numDailies: 3
+    # -- How frequently to look to see if we have new images to prepull (s)
+    pollInterval: 60
+    # -- Maximum time to wait to start a prepuller image (s)
+    pullTimeout: 600
+  form:
+    # -- options form template text keyed by group name.
+    # "default" is always present.
+    forms:
```

Comment:
I don't think this should be a config option, just a template shipped inside the package. This isn't the sort of thing that I would want to stuff into a Helm chart values file and have to document the API, all the substitution parameters, etc.

Reply:
But it is very much the sort of thing that will vary at different installations.